src/ListBox/listBoxOption.css

```diff
@@ -1,42 +1,103 @@
-@import '../proto/definitions/_colors.css';
+@import '../proto/definitions/variables.css';
 
 .default
 {
-    display:    flex;
-    margin:     0;
-    padding:    1rem;
+    display:        flex;
+    align-items:    center;
+    margin:         0;
 
-    margin:     0;
-    padding:    1rem;
+    padding-left:           var( --spacing-2 );
+    padding-right:          var( --spacing-2 );
+    padding-top:            var( --spacing-1 );
+    padding-bottom:         var( --spacing-1 );
 
     cursor:     pointer;
 
+    &:hover,
+    &:hover .optionText,
+    &:hover .description
+    {
+        background-color:   var( --PC-BLUE );
+        color:              var( --PC-WHITE );
+    }
+
     .icon
     {
         flex:           0 0 auto;
-        margin-right:   1rem;
+        margin-right:   var( --spacing-2 );
+    }
+
+    &:hover .icon{
+        fill:           var( --PC-WHITE );
+    }
+
+    .textContainer
+    {
+        flex:           1 0 0%;
+        min-width:      0;
     }
 
-    .text
+    &.disabled,
+    &.disabled .optionText,
+    &.error .disabled
     {
-        flex:       1 0 0%;
-        min-width:  0;
+        pointer-events:             none;
+        cursor:                     auto;
+        color:                      var( --PC-GREY--L40 );
+
+        .description
+        {
+            color:                  var( --PC-GREY--L55 );
+        }
```

Comment:
Again I think this is too vague. Something like this would be better:
```
&.disabled
{
    pointer-events: none;
    cursor:         auto;

    .optionText
    {
        color: var( --PC-GREY--L40 );
    }

     .description
    {
        color: var( --PC-GREY--L55 );
    }
}
```

(That `error` class is not used here and can be removed)